composer.json

```diff
@@ -15,13 +15,13 @@
         }
     ],
     "require": {
-        "php": ">=7.1",
+        "php": "^7.1",
         "symfony/http-kernel": "^3.2|^4.0"
     },
     "require-dev": {
         "symfony/dependency-injection": "^3.2|^4.0",
         "phpunit/phpunit": "^5.6.4",
-        "friendsofphp/php-cs-fixer": "^1.12.4",
+        "friendsofphp/php-cs-fixer": "^1.12.4|^2.0",
```

Comment:
Because this is only for dev purposes, does not have sense to maintain both versions, as we'll only use one of them, and none other dependencies uses that one. We should keep only the second part :)